python/ray/util/sgd/v2/backends/backend.py

```diff
@@ -241,8 +241,11 @@ class InactiveWorkerGroupError(Exception):
 
 class InactiveWorkerGroup():
     # TODO: fix inheritence. perhaps create WorkerGroupInterface.
-    def __getattribute__(self, *args, **kwargs):
-        raise InactiveWorkerGroupError()
+    def __getattr__(self, name):
```

Comment:
could you elaborate more on this change?

Reply:
Yeah, I was testing something earlier and accidentally left the code with `getattr`; just updated the PR to undo that. 

On `__getattr__` vs. `__getattribute__` after reading about it some more I _think_ we should be overriding `__getattr__` in case there are some built-in attributes that still need to be accessed. 

[This StackOverflow answer](https://stackoverflow.com/a/44991681) gives a good overview on how the three are related.